components/ingest-service/backend/elastic/mappings/node_state.go

```diff
@@ -42,11 +42,14 @@ var nodeProps = `
 	},
 	"node_name": {
 		"type": "keyword",
-		"normalizer": "case_insensitive",
 		"fields": {
 			"engram" : {
 				"type": "text",
 				"analyzer": "autocomplete"
+			},
```

Comment:
For each filterable term, a new field is created that will allow matching in the lower case.